scripts/generate_test_code.py

```diff
@@ -762,7 +762,7 @@ def escaped_split(inp_str, split_char):
         raise ValueError('Expected split character. Found string!')
     out = re.sub(r'(\\.)|' + split_char,
                  lambda m: m.group(1) or '\n', inp_str,
-                 len(inp_str)).split('\n')
+                 count = len(inp_str)).split('\n')
```

Comment:
```suggestion
                 count=len(inp_str)).split('\n')
```

otherwise pylint will complain

Reply:
Until we have a CI in the framework repository, please at least run directly relevant tests. Here, at least `check-python-files.sh` (in a CI-like venv), and `make tests`.